data/items/attributes.asm

```diff
@@ -6,6 +6,8 @@ item_attribute: MACRO
 ENDM
 
 ItemAttributes:
+; BUG: Dragon Scale, not Dragon Fang, boosts Dragon-type moves (see docs/bugs_and_glitches.md)
```

Comment:
Please keep BUG comments closer to where they're relevant. In this case, above `; DRAGON_FANG`.